script/staging/FullStagingDeploy.s.sol

```diff
@@ -248,146 +312,154 @@ contract FullStagingDeploy is Script {
+        RateLimitHelpers.setRateLimitData(mainnetController_.LIMIT_USDE_BURN(),      rateLimits, rateLimitData18,    "usdeBurnData",      18);
+        RateLimitHelpers.setRateLimitData(mainnetController_.LIMIT_SUSDE_COOLDOWN(), rateLimits, unlimitedRateLimit, "susdeCooldownData", 18);
+
+        // 4626 and AAVE deposit/withdraw rate limits
+        RateLimitHelpers.setRateLimitData(ausdcDepositKey,  rateLimits, rateLimitData6,  "ausdcDepositData",  6);
+        RateLimitHelpers.setRateLimitData(ausdcWithdrawKey, rateLimits, rateLimitData6,  "ausdcWithdrawData", 6);
+        RateLimitHelpers.setRateLimitData(ausdsDepositKey,  rateLimits, rateLimitData6,  "ausdsDepositData",  18);
+        RateLimitHelpers.setRateLimitData(ausdsWithdrawKey, rateLimits, rateLimitData6,  "ausdsWithdrawData", 18);
+        RateLimitHelpers.setRateLimitData(susdeDepositKey,  rateLimits, rateLimitData18, "susdeDepositData",  18);
+        RateLimitHelpers.setRateLimitData(susdeWithdrawKey, rateLimits, rateLimitData18, "susdeWithdrawData", 18);
+        RateLimitHelpers.setRateLimitData(susdsDepositKey,  rateLimits, rateLimitData18, "susdsDepositData",  18);
+        RateLimitHelpers.setRateLimitData(susdsWithdrawKey, rateLimits, rateLimitData18, "susdsWithdrawData", 18);
+    }
 
-        MainnetControllerInit.subDaoInitFull({
-            addresses: MainnetControllerInit.AddressParams({
-                admin         : mainnet.admin,
-                freezer       : mainnet.config.readAddress(".freezer"),
-                relayer       : mainnet.config.readAddress(".relayer"),
-                oldController : address(0),
-                psm           : psm,
-                vault         : vault,
-                buffer        : buffer,
-                cctpMessenger : mainnet.config.readAddress(".cctpTokenMessenger"),
-                dai           : dai,
-                daiUsds       : daiUsds,
-                usdc          : usdc,
-                usds          : usds,
-                susds         : susds
-            }),
-            controllerInst: instance,
-            data: MainnetControllerInit.InitRateLimitData({
-                usdsMintData         : rateLimitData18,
-                usdsToUsdcData       : rateLimitData6,
-                usdcToCctpData       : unlimitedRateLimit,
-                cctpToBaseDomainData : rateLimitData6,
-                susdsDepositData     : rateLimitData18
-            }),
-            mintRecipients: mintRecipients
+    function _setBaseControllerRateLimits(address rateLimits) internal {
+        RateLimitData memory rateLimitData18 = RateLimitData({
+            maxAmount : USDC_UNIT_SIZE * 1e12 * 5,
+            slope     : USDC_UNIT_SIZE * 1e12 / 4 hours
+        });
+        RateLimitData memory rateLimitData6 = RateLimitData({
+            maxAmount : USDC_UNIT_SIZE * 5,
+            slope     : USDC_UNIT_SIZE / 4 hours
+        });
+        RateLimitData memory unlimitedRateLimit = RateLimitData({
```

Comment:
Same here I would add this as a function to be called, `RateLimitHelpers.unlimitedRateLimitData()` or similar.

Reply:
Just calling directly